src/components/Dashboard/Dashboard.js

```diff
@@ -20,8 +21,71 @@ export const postList = [
     link: `http://localhost:3000/dashboard`,
   },
 ];
+export const kpiList = [
```

Comment:
@cbernat this way it's very hard to use the data, instead, could be returned this way: 

```
{
    'Campaings': [{
        kpiTitle: `Total de Envios`,
        kpiValue: `21458`,
        iconClass: `deliveries`,
        kpiPeriod: `ULTIMOS 30 DIAS`,
        active: true,
    }, {
        kpiTitle: `Tasa de Apertura`,
        kpiValue: `57%`,
        iconClass: `open-rate`,
        kpiPeriod: `ULTIMOS 30 DIAS`,
        active: true,
    }, 
    ...],
    Subscribers: [{...}, {...}, {...}, ...],
}
```

Reply:
`kpiList` & `postList` should be in the `state` of Dashboard

Reply:
I agree with the state part kpiList & postList should be in the state, the will later on be modified by an api call. The other part by with being an array or Campaigns and Subscribers or using it as a type I have no strong opinion.